guides/common/modules/proc_adding-a-remote-console-connection-for-a-managed-host-on-proxmox.adoc

```diff
@@ -0,0 +1,57 @@
+[id="Adding_a_Remote_Console_Connection_for_a_Managed_Host_on_Proxmox_{context}"]
+= Adding a Remote Console Connection for a Managed Host on Proxmox
+
+Use this procedure to add a remote console connection using VNC for a managed host on Proxmox.
+
+.Limitation
+* This procedure only works for hosts based on `qemu`.
+It does not work for LXC container on Proxmox.
+
+.Procedure
+. On your Proxmox, open the required port for VNC:
++
+[options="nowrap", subs="+quotes,verbatim,attributes"]
+----
+# firewall-cmd --add-port=_My_Port_/tcp --permanent
+----
++
+The port is the sum of `5900` and the VMID of your managed host.
+You can view the VMID on the *VM* tab for your managed host in the {ProjectWebUI}.
+If your managed host has the VMID 142, you have to open the port `6042` on your Proxmox.
```

Comment:
```suggestion
For example, if your managed host has the VMID 142, open the port `6042` on your Proxmox.
```

Or at least it looks like an example to me.

Reply:
Applied.